cli/utilities/utils.py

```diff
@@ -389,14 +389,20 @@ def get_service_id(node, service_name):
     Returns the service id of a given service
     Args:
         node (ceph): Node to execute the cmd
-        service_name: Service name
+        service_name: Service name (mon/osd/mgr etc)
 
-    Returns (str): Service ID
+    Returns (str/list): Service ID /ID's
     """
-    out, err = node.exec_command(cmd=f"systemctl --type=service | grep {service_name}")
+    out, err = node.exec_command(
+        cmd=f"systemctl --type=service | grep ceph-{service_name}"
+    )
     if err:
         return None
-    return out.split(" ")[0]
+
+    service_ids = []
+    for item in out.split("\n"):
+        service_ids.append(item.strip().split(" ")[0])
+    return service_ids[:-1]  # [:-1] To ignore the trailing empty entry
```

Comment:
Will there always be a empty string entry at the end?

Can u try with to strip the out value and then do split ?

Reply:
Noted, Strip worked.  Updated